### DISCUSSION TITLE
How to deal with hundreds of mistagged titles

### DISCUSSION THREAD

**Discussion by Whenesday** [Difegue/LANraragi#817] [Q&A]
Hello, I mainly download titles from E-H, and host them in my Synology NAS with LANraragi.

Due to E-H constantly temp-banning me before it can finish tagging my collection with the E-H plugin (and also kept unable to tag them due to id), I tried the Hitomi plugin a while ago. Even though the tagging went smoothly, it instead mis-tags all those titles into various "galleries" groups instead.

I'm looking for a less painful way to untag/initiate tagging with the E-H plugin on those mass affected titles, but I don't see any.

I'm running LANraragi 0.8.81

**Comment by Difegue** [comment]
👋 Sadly I don't really have a way to wipe metadata on selected files, although you might have some luck writing **tag rules** to replace those mistags with blank strings, then running a batch job to apply tag rules to every file?    

I keep referencing #570 but I think that'd help for cases like this.

**Reply by Whenesday** [reply]
Thanks for the response!

Yes, perhaps having plugins's scope that can be defined based on filters can be of some use in this instance.

I'm not exactly sure how utilising the current tag rules feature can help me here, as the mistagging does include common tags which is used legitimately in other titles. 

Unless the intent is to nuke tags indiscriminately, and simply utilise the intended plugin to scrape the entire collection again? 

If so, and if there isn't any way to detect which titles has already been scraped or filters to scope the plugins, I'll just keep running into the same issue of running the plugin until X amount of entries which will initiate the temp ban, and I have no way of resuming progress when the ban is lifted.
